### ISSUE TITLE
Add platforms section on community page

### ISSUE THREAD

**Issue by starbelly** [erlef/website#238]
We need a platforms section on the community page. Immediately grisp and nerves come to mind and naturally would be included. Are there any other platforms I'm not thinking of?

**Comment by paulo-ferraz-oliveira** [comment]
@stritzinger should know of some, I guess.

**Comment by starbelly** [comment]
Yeah, basically this section will be some what of condensed version of https://erlef.org/wg/embedded/content/landscape but there may be other platforms we're not thinking about.

**Comment by paulo-ferraz-oliveira** [comment]
Ah, yeah, that one's a bit hidden, I guess. (at the same time we can't have EVERYthing in the main page, so yeah)
